src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
```

Comment:
[All magic numbers in your code should be constants.](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants)

Reply:
not fixed